### PR TITLE
Smoke me/jkeenan/prepare io socket ip synch 20230816

### LINKED ISSUE #21383: IO-Socket-IP: t/16v6only.t fails during sync-with-cpan
When I install Perl 5 blead (say, at d91c117aa9) and then install IO-Socket-IP-0.42 (the latest version) from CPAN, all tests PASS.
```
Running Build test for PEVANS/IO-Socket-IP-0.42.tar.gz
t/00use.t ........................... ok
t/01local-client-v4.t ............... ok
t/02local-server-v4.t ............... ok
t/03local-cross-v4.t ................ ok
t/04local-client-v6.t ............... ok
t/05local-server-v6.t ............... ok
t/06local-cross-v6.t ................ ok
t/10args.t .......................... ok
t/11sockopts.t ...................... ok
t/12port-fallback.t ................. ok
t/13addrinfo.t ...................... ok
t/14fileno.t ........................ ok
t/15io-socket.t ..................... ok
t/16v6only.t ........................ ok
t/17gai-flags.t ..................... ok
t/18fdopen.t ........................ ok
t/19no-addrs.t ...................... ok
t/20subclass.t ...................... ok
t/21as-inet.t ....................... ok
t/22timeout.t ....................... ok
t/30nonblocking-connect.t ........... ok
t/31nonblocking-connect-internet.t .. ok
t/99pod.t ........................... skipped: Test::Pod 1.00 required for testing POD
All tests successful.
Files=23, Tests=224,  1 wallclock secs ( 0.03 usr  0.01 sys +  0.66 cusr  0.11 csys =  0.81 CPU)
Result: PASS
(./Build test exited with 0)
...
PASS IO-Socket-IP-0.42 x86_64-linux 6.2.0-26-generic
...

  PEVANS/IO-Socket-IP-0.42.tar.gz
  ./Build test -- OK
Running Build install for PEVANS/IO-Socket-IP-0.42.tar.gz
Building IO-Socket-IP
Installing /home/jkeenan/testing/blead/lib/perl5/site_perl/5.39.2/IO/Socket/IP.pm
  PEVANS/IO-Socket-IP-0.42.tar.gz
  ./Build install  -- OK
```
However, when, in anticipation of an Aug 20 release of Perl-5.39.2, I configure blead and then say:
```
perl Porting/sync-with-cpan IO::Socket::IP
```
... I get a failure in `cpan/IO-Socket-IP/t/16v6only.t`:
```
$ cd t;./perl harness -v ../cpan/IO-Socket-IP/t/16v6only.t; cd -
Use of uninitialized value $IO::Socket::errstr in string at t/16v6only.t line 44.

#   Failed test 'Socket creation fails with connection refused'
#   at t/16v6only.t line 47.
#                   ''
#     doesn't match '(?^u:Connection\ refused)'

ok 1 - IPV6_V6ONLY is 1 on $listensock
Use of uninitialized value $IO::Socket::errstr in string at t/16v6only.t line 73.
ok 2 - Unable to connect PF_INET socket to PF_INET6 socket with V6Only true
not ok 3 - Socket creation fails with connection refused
ok 4 - IPV6_V6ONLY is 0 on $listensock
ok 5 - Connected PF_INET socket to PF_INET6 socket with V6Only false
# Looks like you failed 1 test of 6.
ok 6 - Test socket connected to correct peer port
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/6 subtests 

Test Summary Report
-------------------
../cpan/IO-Socket-IP/t/16v6only.t (Wstat: 256 (exited 1) Tests: 6 Failed: 1)
  Failed test:  3
  Non-zero exit status: 1
Files=1, Tests=6,  0 wallclock secs ( 0.00 usr  0.00 sys +  0.02 cusr  0.00 csys =  0.02 CPU)
Result: FAIL
```
(I may have mentioned this problem last month, but I can't find a record of that discussion.)

Any suggestions?

### PR DESCRIPTION
**Pull request by jkeenan**
Should fix https://github.com/Perl/perl5/issues/21383

### CODE REVIEW

File: Porting/Maintainers.pl [hunk 1]
```diff
@@ -677,10 +677,6 @@ package Maintainers;
         'EXCLUDED'     => [
             qr{^examples/},
         ],
-        'CUSTOMIZED' => [
```
Comment (tonycoz) [inline Porting/Maintainers.pl:680]: The entry in `t/porting/customized.dat` also needs to be removed.

Comment (jkeenan) [reply Porting/Maintainers.pl:680]: Hmm ... that suggests that `make test_porting` failed to detect the need to remove that entry.

And when I ran `cd t; ./perl -I../lib porting/customized.t --regen; cd -`, I discovered that `make test_porting` had similarly failed to report the need to modify the database after d91c117aa9f last night.

Comment (jkeenan) [reply Porting/Maintainers.pl:680]: @tonycoz, customized.dat has been updated; can you re-review? Thanks.

**Review by leonerd — APPROVED** [review]
LGTM

**Pull request merged**